shelley/chain-and-ledger/formal-spec/incentives.tex

```diff
@@ -636,9 +648,29 @@ \subsection{How Rewards are returned to Owners and Delegators}
 \label{sec:rewards-return}
 
 In the Incentivised Testnet, rewards will be calculated following the end of the epoch,
-and returned to wallets as part of the transition to the MainNet.
-\khcomment{JC/MG to expand.}
-\khcomment{TODO: Verify that the Testnet rewards will not be added to stake pots. Still not clear from the Testnet documentation.}
+and returned to wallets as part of the transition to the MainNet.  This will be achieved
+using the instantaneous rewards return mechanism that is described below.
+
+\begin{itemize}
+  \item
+    When the testnet terminates, a snapshot will be taken in order to calculate the rewards that were earned.
```

Comment:
please consistently use a variant, maybe make TestNet and MainNet a latex macro?

Reply:
OK.